miso-web/src/main/webapp/scripts/lims.js

```diff
@@ -148,6 +148,74 @@ var Utils = Utils || {
       }
     }
   },
+  
+  showOkDialog: function(title, fields) {
+    var dialogArea = document.getElementById('dialog');
+    while (dialogArea.hasChildNodes()) {
+      dialogArea.removeChild(dialogArea.lastChild);
+    }
+  
+    var output = {};
+    fields.forEach(function(field) {
+      var p = document.createElement('P');
+      p.textContent = field;
+      dialogArea.appendChild(p);
+    });
+    
+    var dialog = jQuery('#dialog').dialog({
+      autoOpen: true,
+      width: 500,
+      title: title,
+      modal: true,
+      buttons: {
+        'OK': {
+          id: 'ok',
+          text: 'OK',
+          click: function () {
+            dialog.dialog("close");
+          }
+        }
+      }
+    });
+  },
+  
+  showConfirmDialog: function(title, okButton, fields, callback) {
+      var dialogArea = document.getElementById('dialog');
+      while (dialogArea.hasChildNodes()) {
+          dialogArea.removeChild(dialogArea.lastChild);
+      }
+      
+      var output = {};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'output' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)